app/controllers/auth/Auth.controller.ts

```diff
@@ -65,19 +66,21 @@ export class AuthController {
 
     public static async verify(request: Request, response: Response) {
         const { key } = request.query;
-        const redirectUrl = `${process.env.FRONT_URL}`;
 
         const foundToken = await EmailVerification.findOne({ where: { token: key } });
 
-        if (!foundToken) {
+        if (!_.isNil(foundToken)) {
```

Comment:
Revert back to `!foundToken`, token can't be undefined so lodash is not needed

Reply:
```typescript
  /**
     * Finds first entity that matches given options.
     */
    static findOne<T extends BaseEntity>(this: ObjectType<T>, options?: FindOneOptions<T>): Promise<T | undefined>;
```

It can be undefined.

Reply:
Additionally lodash is designed to help with edge cases in which !identifer will not cover. (i'm not saying its valid in this case but helpful).

https://jsbin.com/suhineziva/2/edit?js,console

![image](https://user-images.githubusercontent.com/12329422/58342867-858f8a80-7e49-11e9-8c7c-d363d672883a.png)

User validation is important, you cannot enforce the client to send the token.